cg/io/txt.py

```diff
@@ -33,4 +36,7 @@ def concat_txt(
     for file_path in file_paths:
         file_content: str = read_txt(file_path, read_to_string=True)
         content += f"{file_content}\n"
-    write_txt(content=content, file_path=target_file)
+    if target_file:
+        write_txt(content=content, file_path=target_file)
```

Comment:
Ideally `concat_txt` does only that and instances where such txt needs to be written a different function called `write_concat_txt` is called.

`write_concat_txt` then performs `concat_txt` and `write_txt`

Reply:
done